### ISSUE TITLE
Resource temporarily unavailable (api.nuget.org:443)

### ISSUE THREAD

**Issue by victron** [QuantConnect/lean-cli#122]
I trying to run `lean live <my project>` using 'Interactive Brokers' data feed.
But getting an error like this
```
error: Unable to load the service index for source https://api.nuget.org/v3/index.json.
error:   Resource temporarily unavailable (api.nuget.org:443)
error:   Resource temporarily unavailable

Usage: NuGet.CommandLine.XPlat.dll package add [options]
......
```
In internet there is some advice to configure proxy for `NuGet`, then it means something in docker image. Please advice what is a problem

my environment:
- running inside WSL on Ubuntu 22.04.1 LTS
- lean, version 1.0.97
- Python 3.10.4

**Comment by Martin-Molinero** [comment]
Hey @victron! Sound like nuget.org has some down time. Is this still happening?

**Comment by victron** [comment]
no, it's not  nuget.org problem. curl response is ok. I tested on hardware pc with debian and it's ok. So, may be some relation to ubuntu or wsl itself. Can you point me to some doc to understand what inside your docker image?
btw, may be it's not a good idea to download something from 3-d party, in my understanding everything should be in image as minimum for security reasons.

**Comment by Martin-Molinero** [comment]
> btw, may be it's not a good idea to download something from 3-d party, in my understanding everything should be in image as minimum for security reasons.

It's pulling lean packages and dependencies, same is done for any custom for libraries I believe

> In internet there is some advice to configure proxy for NuGet

Does the host machine have a proxy setup? Sounds like that might be missing for the child docker container maybe

**Comment by jaredbroad** [comment]
We have a reproducible case [here](https://app.intercom.com/a/inbox/9a05faa80b1df601897603e983f2c594391bd0a0/inbox/shared/all/conversation/1640600681672?view=List). Rahul can you please if you have a proxy?

**Comment by Adroit-Algos** [comment]
The machine has access to direct internet. I tried Wget https://api.nuget.org/v3/index.json and it works fine. I think the issue is with the docker file.
